samples/zigbee/shell/README.rst

```diff
@@ -0,0 +1,166 @@
+.. _zigbee_shell_sample:
+
+Zigbee: Shell
+#############
+
+.. contents::
+   :local:
+   :depth: 2
+
+This :ref:`Zigbee <ug_zigbee>` shell sample demonstrates a Zigbee router (with the possibility of being a coordinator) that uses the shell interface inside Zephyr's :ref:`zephyr:shell_api` for interaction.
+
+The sample can be used for several purposes, including:
+
+* Initial configuration of the network - forming a network as a coordinator, adding devices to the network with the install codes, setting the extended PAN ID.
+* Benchmarking - measuring time needed for message to travel from one node to another.
+
+Available CLI interfaces
+************************
+
+The CLI can be accessed by using the following interfaces:
+
+* UART
+* RTT
+
+These interfaces are completely independent one from another and can be used simultaneously or exclusively.
+
+The sample demonstrates the usage of commands listed on :ref:`zigbee_cli_reference`.
+Zigbee CLI is integrated into the system shell that is accessible over the available interfaces.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52840dk_nrf52840, nrf52833dk_nrf52833, nrf5340dk_nrf5340_cpuapp, nrf21540dk_nrf52840
+
+You can use two or more of the development kits listed above and mix different development kits.
+
+Additionally, you can use this sample with any other :ref:`Zigbee <ug_zigbee>` sample application for testing the communication or other sample-specific functionalities.
+
+Overview
+********
+
+The Zigbee shell sample takes the Zigbee Router role and implements two clusters (Basic and Identify) that used to be required by the Zigbee Home Automation profile.
+The Basic cluster provides attributes and commands for determining basic information about the node.
+The Identify cluster allows to put the device into the identification mode, which provides a way to locate the device.
+The device also includes all Zigbee shell commands, that allow for discovering, controlling and testing other Zigbee devices.
+
+User interface
+**************
+
+LED 3:
+    Turns on when the device joins the network.
+
+LED 4:
+    Blinks to indicate that the identification mode is on.
+
+Button 4:
+    Toggles the identification mode on the device.
+
+All other interactions with the application can be handled using serial communication.
+See :ref:`zigbee_cli_reference` for the list of available serial commands.
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/zigbee/shell`
+
+|enable_zigbee_before_testing|
+
+.. include:: /includes/build_and_run.txt
+
+.. _zigbee_shell_sample_testing:
+
+Testing
+=======
+
+After building the sample and programming it to your development kits, test it by performing the following steps:
+
+#. Turn on the development kits.
```

Comment:
```suggestion
#. Turn on the development kits.
#. |connect_kit|
```
This will expand to:
`Connect the kit to the computer using a USB cable. The kit is assigned a COM port (Windows) or ttyACM device (Linux), which is visible in the Device Manager.`